recipients.go

```diff
@@ -0,0 +1,70 @@
+package updown
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// Recipients represents a end point type called by Updown on any event
+type Recipient struct {
+	ID    string `json:"id,omitempty"`
```

Comment:
here as well, to keep it consistent, I reckon we should use an uppercase on the first letter only.